### PR TITLE
#472 : Autorized users for access to configuration service 

### PR DESCRIPTION
**Pull request by anandrgitnirman**
#472

### CODE REVIEW

File: configuration_service/configuration_service.go [hunk 1]
```diff
@@ -78,61 +96,51 @@ func (service ConfigurationService) IsDaemonProcessingRequests(ctx context.Conte
...
 
+	signerFromMessage, err := authutils.GetSignerAddressFromMessage(service.getMessageBytes(prefix, auth.CurrentBlock), auth.GetSignature())
+	if err != nil {
+		log.Error(err)
+		return err
+	}
 	//Check if the Signature is Valid and Signed accordingly
-	if err = authutils.VerifySigner(service.getMessageBytes(prefix, auth.CurrentBlock),
-		auth.GetSignature(), blockchain.HexToAddress(service.address)); err != nil {
+	if err = service.checkAuthenticationAddress(*signerFromMessage); err != nil {
```
Comment (raamb) [inline configuration_service/configuration_service.go]: Do we not need to verifySigner anymore?

File: configuration_service/configuration_service.go [hunk 2]
```diff
@@ -78,61 +96,51 @@ func (service ConfigurationService) IsDaemonProcessingRequests(ctx context.Conte
...
-	}
-
 	return service
 }
 
-//Message format has been agreed to be as the below ( prefix,block number,and authenticating address)
-func (service *ConfigurationService) getMessageBytes(prefixMessage string, blocknumber uint64) []byte {
+//Message format has been agreed to be as the below ( prefix,block number,and authenticating authenticationAddressList)
+func (service *ConfigurationService) getMessageBytes(prefixMessage string, blockNumber uint64) []byte {
 	message := bytes.Join([][]byte{
```
Comment (raamb) [inline configuration_service/configuration_service.go]: service.address is not used in the message?

File: configuration_service/configuration_service.go (hunk 1 above)
Comment (anandsnet) [reply configuration_service/configuration_service.go]: We do, just that it's not one address but a list of addresses that we need to loop and check.hence a different method

File: configuration_service/configuration_service.go (hunk 2 above)
Comment (anandsnet) [reply configuration_service/configuration_service.go]: Not required , you derive the address from the signature ,

**Pull request merged**